include/ola/rdm/RDMAPI.h

```diff
@@ -172,6 +172,28 @@ struct clock_value_s {
 
 typedef struct clock_value_s ClockValue;
 
+/*
+ * Dimmer info values
+ */
+typedef struct {
+  uint16_t min_level_lower_limit;
+  uint16_t min_level_upper_limit;
+  uint16_t max_level_lower_limit;
+  uint16_t max_level_upper_limit;
+  uint8_t curves_supported;
+  uint8_t resolution;
+  uint8_t split_levels_supported;
```

Comment:
Likewise this should be a bool too as per https://peter-ola-rdm-app.appspot.com/pid/display?manufacturer=0&pid=832

Reply:
Done.